src/components/TimespanForm.js

```diff
@@ -51,6 +51,9 @@ class TimespanForm extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
```

Comment:
@Dananji I think`componentWillReceiveProps` may be a deprecated lifecycle method.  Can you see if `getDerivedStateFromProps` or some other method can give you the timing you need on this?

Reply:
@adamjarling  Yeah, `componentWillReceiveProps` is going to be removed in version 17. I will see there's any other way to do this including `getDerivedStateFromProps`. Should I create a different ticket for this one? I feel like we might need to change some stuff around for this.

Reply:
Yea, if we have the time now to fix, may as well get ahead of it?  I'd go ahead and create the ticket.

Reply:
Created issue to fix this, #16